benchmark/build.gradle.kts

```diff
@@ -9,7 +9,7 @@ repositories {
 
 java {
     toolchain {
-        languageVersion.set(JavaLanguageVersion.of(11))
+        languageVersion = JavaLanguageVersion.of(21)
```

Comment:
The GitHub workflows probably need a JDK update to support the latest java for testing and benchmarking